docker/wa/src/main/resources/application.properties

```diff
@@ -26,8 +26,17 @@ spring.http.encoding.force=true
 
 server.servlet.contextPath=/syncope-wa
 
-spring.resources.static-locations=classpath:/syncope/static,classpath:/static
-spring.thymeleaf.templateResolverOrder=1 
+spring.resources.static-locations=classpath:/thymeleaf/static,classpath:/static
```

Comment:
Can you please check that
* `docker/wa/src/main/resources/application.properties`
* `docker/wa/src/main/resources/wa.properties`

are aligned with

* `wa/starter/src/main/resources/application.properties`
* `wa/starter/src/main/resources/wa.properties`

?